src/vtol.py

```diff
@@ -101,14 +74,13 @@ def start_auto_mission(self):
         self.commands.next = 0
         self.mode = VehicleMode("AUTO")
 
-        if self.configs["vehicle_type"] == "Quadcopter":
-            msg = self.message_factory.command_long_encode(
-                0, 0,    # target_system, target_component
-                mavutil.mavlink.MAV_CMD_MISSION_START, #command
-                0, #confirmation
-                0, 0, 0, 0, 0, 0, 0)    # param 1 ~ 7 not used
-            # send command to vehicle
-            self.send_mavlink(msg)
+        msg = self.message_factory.command_long_encode(
+            0, 0,    # target_system, target_component
```

Comment:
why are these two parameters zero? I see you refer to them as target_system and target_component is there any point in which these parameters would not be zero? if so they should be assigned to variables.

Reply:
Again, this is from arupilot, I'm not particularly concerned about these for now